miasm2/jitter/jitload.py

```diff
@@ -449,7 +469,8 @@ def handle_lib(jitter):
 
     def handle_function(self, f_addr):
         """Add a brakpoint which will trigger the function handler"""
-        self.add_breakpoint(f_addr, self.handle_lib)
+        if self.get_breakpoint(f_addr) is None:
```

Comment:
Here, if a breakpoint is already set, this method will silently failed, which is probably misleading for the user and hardly debuggable after